DICOM/DCMSUM.pm

```diff
@@ -340,43 +340,47 @@ QUERY
           (
            TarchiveID, SeriesNumber,      FileNumber, 
            EchoNumber, SeriesDescription, Md5Sum, 
-           FileName
+           FileName,   TarchiveSeriesID
           ) 
         VALUES 
           (
            ?,          ?,                 ?, 
            ?,          ?,                 ?, 
-           ?
+           ?,          ?
           )
 QUERY
+    my $query_select_TarchiveSeriesID = "SELECT TarchiveSeriesID FROM tarchive_series WHERE SeriesUID = ? AND EchoTime= ?";
+    my $select_TarchiveSeriesID = $dbh->prepare($query_select_TarchiveSeriesID);
     my $insert_file = $dbh->prepare($insert_query);
     my $dcmdirRoot = dirname($self->{dcmdir});
     foreach my $file (@{$self->{'dcminfo'}}) {
         # insert the file
         my $filename = $file->[4];
         $filename =~ s/^${dcmdirRoot}\///;
         $file->[2] = undef if($file->[2] eq '');
+        $select_TarchiveSeriesID->execute($file->[24], $file->[6]);
```

Comment:
might be good to document in a comment what is $file->[24], $file->[6]. I am guessing it will be SeriesUID and EchoTime but it would be better to document.